### PR TITLE
Issues/19

### LINKED ISSUE #19: GET department/{id}/memberUnits returns duplicate results
An request to fetch the member units for a given department can return duplicate unit results. I noticed this on the [test server page for UA-VPIT](https://itpeople-test.iu.edu/departments/2547).  

**Request**

`GET https://api.itpeople-test.iu.edu/departments/2547/memberUnits`

**Response**

image

**Page view**

image

### PR DESCRIPTION
**Pull request by jasonfrancis**
Resolves #19 - duplicate Units for requests to `departments/{id}/memberUnits`.
Much like #14 it was simply missing a Distinct clause.

### CODE REVIEW

File: tests/API/Integration/DepartmentsTests.cs [hunk 1]
```diff
@@ -99,6 +99,49 @@ public async Task GetAuditorParksMemberUnits()
...
+					NameLast = "Traeger",
+					Position = "Sr. Auditor",
+					Location = "Pawnee",
+					Campus = "Pawnee",
+					CampusPhone = "5-6789",
+					CampusEmail = "ctraeger@auditz.com",
+					Notes = "",
+					PhotoUrl = "",
+					Responsibilities = Responsibilities.None,
+					DepartmentId = TestEntities.Departments.AuditorId,
```
Comment (vbhupathi) [inline tests/API/Integration/DepartmentsTests.cs:122]: I think its `DepartmentId = TestEntities.Departments.Auditor.Id`?

File: tests/API/Integration/DepartmentsTests.cs [hunk 2]
```diff
@@ -99,6 +99,49 @@ public async Task GetAuditorParksMemberUnits()
...
+					UnitMemberships = new List<UnitMember>()
+				};
+				
+				var chrisAuditor = new UnitMember
+				{
+					UnitId = TestEntities.Units.AuditorId,
```
Comment (vbhupathi) [inline tests/API/Integration/DepartmentsTests.cs:128]: Same question here `UnitId=TestEntities.Units.Auditor.Id`?  (Sorry, If I'm missing something)

File: tests/API/Integration/DepartmentsTests.cs (hunk 1 above)
Comment (jasonfrancis) [reply tests/API/Integration/DepartmentsTests.cs:122]: They should be the same thing.  `TestEntities.Departments.Auditor.Id` is set using `TestEntities.Departments.AuditorId`.
Feel free to change it if you think it's easier to read the other way.

File: tests/API/Integration/DepartmentsTests.cs (hunk 2 above)
Comment (jasonfrancis) [reply tests/API/Integration/DepartmentsTests.cs:128]: Ditto. `TestEntities.Units.Auditor.Id` is set using `TestEntities.Units.AuditorId`.

Once again, feel free to change it for legibility if you have a preference.

File: tests/API/Integration/DepartmentsTests.cs (hunk 1 above)
Comment (vbhupathi) [reply tests/API/Integration/DepartmentsTests.cs:122]: Thanks, Jason.

File: tests/API/Integration/DepartmentsTests.cs (hunk 2 above)
Comment (vbhupathi) [reply tests/API/Integration/DepartmentsTests.cs:128]: Thanks, again.

**Review by vbhupathi — APPROVED** [review]
Good fix👍

**Review by jhoerr — APPROVED** [review]

**Pull request merged**